ecosystem/sep-0010.md

```diff
@@ -18,57 +18,58 @@ This SEP defines the standard way for clients such as wallets or exchanges to cr
 
 This protocol is a variation of mutual challenge-response, which uses Stellar transactions to encode challenges and responses.
 
-It involves three components:
-- A domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`), referred to as the home domain.
-- An endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on the home domain, a sub-domain of the home domain, or any other domain. The domain where the endpoint is hosted is referred to as the web auth domain. 
-- A client who will authenticate using a Stellar account.
+It involves the following components:
+- A **Home Domain**: a domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`).
+- A **Server**: a server providing the `WEB_AUTH_ENDPOINT` that implements the GET and POST operations discussed in this document. The server's domain may be the **Home Domain**, a sub-domain of the **Home Domain**, or any other domain.
```

Comment:
💡 This is sort of out-of-scope of your PR but I think this tiny tweak is worthwhile. The 'any other domain' doesn't quite communicate the same intent as saying it can be a different domain, although they're similar.
```suggestion
- A **Server**: a server providing the `WEB_AUTH_ENDPOINT` that implements the GET and POST operations discussed in this document. The server's domain may be the **Home Domain**, a sub-domain of the **Home Domain**, or a different domain.
```